aws-redis-cluster.yml

```diff
@@ -133,6 +132,6 @@ examples:
   bind_params: {}
 - name: with-custom-node_type
   description: Create a Redis instance with a custom node_type
-  plan_id: c7f64994-a1d9-4e1f-9491-9d8e56bbf146
+  plan_id: 2deb6c13-7ea1-4bad-a519-0ac9600e9a29
```

Comment:
issue: this is an old plan id. This change has to be reverted

Reply:
I intentionally generated that UUID. Is new.
I couldn't use the previous UUID because [`node_type` is specified in the example-basic-plan (c7f64994...)](https://github.com/pivotal/cloud-service-broker-ci/pull/181/files#diff-e7ddf76429dea95042d3f0b84fcb624494ff53a357178b1cf7e813caf1cfcbf6R1504) and therefore can't be customised as a service-specific configuration parameter.

To ensure that we can continue testing that scenario (creating a new service with a custom `node_type`) I had to create a new plan which doesn't specify a `node_type` as you can see in [integration-tests](https://github.com/cloudfoundry/csb-brokerpak-aws/pull/774/files#diff-6a986ed758df98265a76ea58d4374b103546b7ee077fd77c66fd9a7372c2c192R39) and in [CI repo](https://github.com/pivotal/cloud-service-broker-ci/pull/181/files#diff-e7ddf76429dea95042d3f0b84fcb624494ff53a357178b1cf7e813caf1cfcbf6R1509).

Reply:
Fantastic! We have to add the plan definition to `.envrc` file.

Thank you very much! :rocket: